.github/workflows/push-tests.yml

```diff
@@ -11,6 +11,8 @@ on:
     branches:
```

Comment:
Also I dont understand the need for this workflow.

We dont allow pushes to master branch from the settings, and we generally dont push directly - we always create a PR. I think this workflow can go away - since there is another workflow covering the tests execution on PRs.

Reply:
OK I see from the git history that this was introduced for codecov reasons. 
Do we run codecov on all branches @jkonecny12 ?

Reply:
Yes, we do https://app.codecov.io/gh/rhinstaller/anaconda/tree/rhel-10.